CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributor Onboarding
+
+## General
+
+- Helps start contributions to Durable Functions in JavaScript
+- Helps setup development environment across platforms for Durable Functions in JavaScript
+
+## Pre-reqs
+
+- OS
+    - MacOS (or) Windows10
+- Language Runtimes
+    - .NET Core 2.0
+    - Python 3.6.x
+- Editor
+    - VS Code (or) Visual Studio
```

Comment:
Should we add links to the language runtime installation downloads and to the editors?

Reply:
Good idea, done!